demo-project/package.json

```diff
@@ -3,8 +3,18 @@
     "version": "1.0.0",
     "main": "index.js",
     "license": "MIT",
+    "engines": {
+        "node": ">= v18.17.1"
+    },
     "scripts": {
-        "build": "parcel build main.ts"
+        "build": "parcel build main.ts",
+        "format": "npx prettier --write .",
+        "lint": "yarn lint-backend && yarn lint-frontend",
+        "lint-backend": "eslint --max-warnings 0 --config ./.eslintrc.json  \"./**/*.ts*\"",
+        "lint-frontend": "eslint --max-warnings 0 --config ./.eslintrc.json  \"./demo-frontend/**/*\"",
+        "fix": "yarn fix-backend && yarn fix-frontend",
+        "fix-backend": "eslint --max-warnings 0 --fix --config ./.eslintrc.json  \"./**/*.ts*\"",
```

Comment:
This should be `yarn lint-backend --fix`, right?

Reply:
[d8f68f6](https://github.com/KILTprotocol/opendid/pull/55/commits/d8f68f65beb66003ce6cb05aceb52aa50db830fa)